README.md

```diff
@@ -5,3 +5,15 @@ A little utility that pre-authorizes Instruments to run UIAutomation scripts aga
 
 [![Build Status](https://travis-ci.org/appium/authorize-ios.svg)](https://travis-ci.org/appium/authorize-ios)
 [![Coverage Status](https://coveralls.io/repos/appium/authorize-ios/badge.svg?branch=master&service=github)](https://coveralls.io/github/appium/authorize-ios?branch=master)
+
+##Installation##
+
+(if you're using appium, check the appium readme for how to call this function. This readme is for developers who are using this npm package, not appium users)
```

Comment:
I'm curious about Appium capitalization. I think I recall @jlipps doesn't capitalize it